.github/workflows/ci-pr-main-program.yml

```diff
@@ -5,9 +5,9 @@ on:
     branches:
       - main
```

Comment:
pls aadd staging branches here in order to run cli

Reply:
added